arches/app/etl_modules/base_import_module.py

```diff
@@ -287,10 +286,86 @@ def write(self, request):
             if summary["cumulative_excel_files_size"] / 1000000 > use_celery_file_size_threshold_in_MB:
                 response = self.run_load_task_async(request, self.loadid)
             else:
-                response = self.run_load_task(self.userid, files, summary, result, self.temp_dir, self.loadid)
+                response = self.run_load_task(self.userid, files, summary, result, self.temp_dir, self.loadid, multiprocessing)
 
             return response
 
+    def cli(self, source):
+        initiated = self.start(self.request)
+
+        if initiated["success"]:
+            read = self.read_for_cli(source)
+        else:
+            return {"success": False, "data": {"title": _("Error"), "message": initiated["data"]["message"]}}
+
+        if read["success"]:
+            self.request.POST.__setitem__('load_details', json.dumps({"result": read["data"]}))
+            self.request.POST.__setitem__('multiprocessing', True)
+            written = self.write(self.request)
+        else:
+            return {"success": False, "data": {"title": _("Error"), "message": read["data"]["message"]}}
+
+        if written["success"]:
+            return {"success": True, "data": "done"}
+        else:
+            return {"success": False, "data": {"title": _("Error"), "message": written["data"]["message"]}}
+
+    def read_for_cli(self, source):
+        self.cumulative_excel_files_size = 0
+        self.temp_dir = os.path.join(settings.UPLOADED_FILES_DIR, "tmp", self.loadid)
+        try:
+            self.delete_from_default_storage(self.temp_dir)
+        except (FileNotFoundError):
+            pass
+        file_name = os.path.basename(source).split('/')[-1]
```

Comment:
`os.path.basename` returns the name of a file, so this seems like it should just be 
```
file_name = os.path.basename(source)
```
I could be missing something, but I don't think the `.split('/')[-1]` is doing anything because there shouldn't be any slashes in the file's basename.